modules/microshift-greenboot-dir-structure.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: CONCEPT
+[id="microshift-greenboot-dir-structure_{context}"]
+= How greenboot uses directories to run scripts
```

Comment:
@dhellmann We actually do, for users that are unfamiliar with it. The answer is not to just provide a collection of links. We can add links to RHEL, but basic information as to what this is and does should stay in MicroShift, since this is an an optional component for it. This PR is/was nearly complete, as @ggiguash and I did meet and discuss the content.

Reply:
I don't know where we draw the line. If this is the usual approach, let's move ahead. If the MicroShift team starts receiving support requests for greenboot we may need to revisit.

Reply:
I think we draw the line at giving the essential "what it is" and "how to use it." This PR looks long, but in the scope of docs, it's pretty basic. Docs gets a lot of bugs of the "I don't know what this is" nature. So I think we have opposing forces at work and we have to figure out the narrow path between. I agree, let's see what customer feedback we get and proceed from there.